sort_numbers.py

```diff
@@ -0,0 +1,25 @@
+import sys
+
+
+#This function will sort a given sequence of numbers
+#if value in the list of numbers is not interger or float, program will exit
+def sort_numbers(numbers):
+    for i, value in enumerate(numbers):
+        try:
+            numbers[i] = float(value)
+        except ValueError:
+            print("Error:", value, "is not a valid number.")
+            sys.exit(1)
+    numbers = [num for num in numbers if num >= 0] + [num for num in numbers if num < 0]
```

Comment:
Can you explain to me what this does?

Reply:
I slit the numbers into 2 small lists, the first one stores all numbers >=0, and the other stores all numbers <= 0, after that I add 2 small lists to be a new list containing a both positive and negative numbers.

Reply:
Ah, yes, it is redundant. I do not need it when doing the sort numbers.